cnudie/util.py

```diff
@@ -292,3 +294,29 @@ def enumerate_group_pairs(
             _add_if_not_duplicate(resource_diff.resource_refs_only_right, i)
 
     return resource_diff
+
+
+CTF_CD_FILENAME = 'component-descriptor.yaml'
+
+
+def extract_component_descriptor_from_ctf_archive(
```

Comment:
why not simply yield all component-descriptors? the caller may still decide to not allow / bail out if more than one cd is found. there is no reason not to support more than one cd in this function, methinks

Reply:
naming-nit: I would suggest to simply omit the verb (extract), as it does not really add much semantics to the name